content/learn/book/plugin-development/_index.md

```diff
@@ -0,0 +1,140 @@
++++
+title = "Plugin Development"
+weight = 3
+sort_by = "weight"
+template = "docs-section.html"
+page_template = "docs-section.html"
++++
+
+Bevy has a plug and play architecture, where you can easily add plugins for new features, or use your own plugins instead of the built-in ones. You can also create third party plugins that others may use in their applications.
```

Comment:
```suggestion
Bevy has a plug-and-play architecture, where you can easily add plugins for new features, or use your own plugins instead of the built-in ones. You can also create third party plugins that others may use in their applications.
```